### ISSUE TITLE
Invoke-WebRequest / Invoke-RestMethod -Verbose shows requested rather than actual version

### ISSUE THREAD

**Issue by mklement0** [PowerShell/PowerShell#19615]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

Note: 

* Perhaps the behavior is by design: show what version was _requested_.
* However, at least the _actual_ HTTP version used is _also_ worth knowing.

```powershell
 ((Invoke-WebRequest https://example.org -Verbose -HttpVersion 3).RawContent -split '\r?\n', 2)[0]
```

### Expected behavior

```console
VERBOSE: HTTP/2.0 GET with 0-byte payload
VERBOSE: received response of content type text/html of unknown size
HTTP/2.0 200 OK
```

### Actual behavior

```console
VERBOSE: HTTP/3.0 GET with 0-byte payload
VERBOSE: received response of content type text/html of unknown size
HTTP/2.0 200 OK
```

That is, even though the *actual* HTTP version is 2.0, the -Verbose output shows the *requested* version.

### Environment data

```powershell
PowerShell 7.4.0-preview.3
```

**Comment by CarloToso** [comment]
I think it's by design the first verbose is called `reqVerboseMsg`, but curl has a different behaviour so maybe we should change webcmdlets...

**Comment by mklement0** [comment]
I appreciate your PR, @CarloToso (as I have many previous ones).

Yes, I think it makes more sense to show the _resulting_ version with `-Verbose`, given that as a caller it is reasonable to assume that you _already know_ what version, if any, you requested (though, conceivably, _both_ pieces of information could be shown, but I personally don't think that's necessary).

**Comment by daxian-dbw** [comment]
Closed via #19616
